modules/nf-scil/segmentation/fastseg/meta.yml

```diff
@@ -26,6 +26,11 @@ input:
       description: Nifti T1 volume to segment into tissue maps.
       pattern: "*.{nii,nii.gz}"
 
+  - lesion:
+      type: file
+      description: Nifti lesion volume to correct the white matter with a lesion mask.
```

Comment:
Could you describe how the lesion volume is merged with the white matter mask? It would help clarify how it is handled if someone wants to use the module and isn't familiar with it.